src/frontend/parallelization_phase.nit

```diff
@@ -66,15 +65,38 @@ private class ParallelizationPhase
 			classname += nmethdef.n_methid.as(AIdMethid).n_id.text
 		end
 
-		# Create a string corresponding to the threaded class
-		var s ="""
+		# String corresponding to the generated class
+		var s : nullable String = null
```

Comment:
It looks to me that `s` doesn't need to be nullable, does it? Both bodies in the if construct assign a value to `s`. You don't event need to declare its type!